src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +425,175 @@ public void SetFocus(bool focused)
+        }
+
+        /// <summary>
+        /// Resumes the operation associated with the view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Resume ()
+        {
+            Interop.ChromiumEwk.ewk_view_resume(_realHandle);
+        }
+
+        /// <summary>
+        /// Gets the current scale factor of the page.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public double Scale
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_view_scale_get(_realHandle);
+            }
+        }
+
+        /// <summary>
+        /// Sets the current scale factor of the page.
+        /// </summary>
+        /// <param name="scaleFactor">A new level to set.</param>
+        /// <param name="scrollTo">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void SetViewScale (double scaleFactor, Point scrollTo)
+        {
+            Interop.ChromiumEwk.ewk_view_scale_set(_realHandle, scaleFactor, scrollTo.X, scrollTo.Y);
+        }
+
+        /// <summary>
+        /// Sets the current page's visibility.
+        /// </summary>
+        /// <param name="enable">'true' to set on the visibility of the page, 'false' otherwise.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void SetViewVisibility (bool enable)
+        {
+            Interop.ChromiumEwk.ewk_view_visibility_set(_realHandle, enable);
+        }
+
+        /// <summary>
+        /// Get and Sets the scroll position of the page.
+        /// </summary>
+        /// <returns>The class Point object with X, Y coordinates.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Point ScrollPos
+        {
+            get
+            {
+                Point p;
+                Interop.ChromiumEwk.ewk_view_scroll_pos_get(_realHandle, out p.X, out p.Y);
+                return p;
+            }
+            set
+            {
+                Interop.ChromiumEwk.ewk_view_scroll_set(_realHandle, value.X, value.Y);
+            }
+        }
+
+        /// <summary>
+        /// Scrolls the webpage of view by x and y.
+        /// </summary>
+        /// <param name="scrollBy">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ScrollBy (Point scrollBy)
+        {
+            Interop.ChromiumEwk.ewk_view_scroll_by(_realHandle, scrollBy.X, scrollBy.Y);
+        }
+
+        /// <summary>
+        /// Searches and highlights the given text string in the document.
+        /// </summary>
+        /// <param name="text">The text to find.</param>
+        /// <param name="option">The options to find.</param>
+        /// <param name="maxMatchCount">The maximum match count to find, unlimited if 0.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void FindText (string text, FindOption option, int maxMatchCount)
```

Comment:
How to get result?
```suggestion
        public bool FindText (string text, FindOption option, int maxMatchCount)
```

Reply:
User can see the result on the screen, each result will be highlighted.

Reply:
ewk api is returing bool, so how about returing bool?

Reply:
We can return boolean, but @m-macias said "It should be void, in C# we don't return boolean on success."

Reply:
For this particular method I have said "Please rewrite the statement, because the success term is too vague. Please define what the success means in the context of FindText(...) function.", because its purpose is to **Find** something or not. Yes, we don't return boolean on success in C# only if you want to indicate the success as in C code, but of course we can return True/False in the function which answers the questions `Have I found any text?`.

Reply:
Thank you for your detailed answer.
In fact, `FindText` works asynchronously, so the return value means that request has been sent successfully in C code.
In this case, we should not return boolean value in C#, right?